.github/workflows/build.yml

```diff
@@ -25,17 +25,11 @@ jobs:
       - name: Create full source ZIP archive and Signature
         run: |
           zip -9vr daed-full-src.zip . -x .git/\*
-          FILE=./daed-full-src.zip
-          DGST=$FILE.dgst
-          md5sum        $FILE >>$DGST
-          shasum -a 1   $FILE >>$DGST
-          shasum -a 256 $FILE >>$DGST
-          shasum -a 512 $FILE >>$DGST
```

Comment:
Why remove?

Reply:
<img width="640" alt="image" src="https://github.com/daeuniverse/daed/assets/30586220/129086d0-6712-447d-998a-e57112dbd72f">
 Maybe you want to calculate all digests in one step, but I cannot see `daed-full-src.zip.dgst`. Something goes wrong"?

Reply:
https://github.com/daeuniverse/daed/pull/40/commits/c3abeb8c8ac652e965f0512adb835ddb2764ce52 does not fix up the problem `daed-full-src.zip should not be zipped again`.

Reply:
`daed-full-src.zip` is actually zipped by the Action workflow, and when upload it to Action artifacts, it will be zipped again as `daed-full-src.zip.zip`, so in the last release job, we should only  generate the dgst file for it, but not zip it again like web files.
![图片](https://github.com/daeuniverse/daed/assets/50002150/3ccf109b-1e4e-42fa-9ea3-ff9525543402)